internal/lib/constants.go

```diff
@@ -114,6 +114,7 @@ const (
 	NPLService                                 = "NPLService"
 	SyncStatusKey                              = "syncstatus"
 	NoFreeIPError                              = "No available free IPs"
+	AKO_ENCODED                                = "--ako-encoded"
```

Comment:
@monotosh-avi / @akshayhavile should be prefix this value instead of suffixing?
Also can we instead call it - "ako-object-id--"

Reply:
@sudswasavi : Encoded object name currently looks like <cluster-name>--encodedvalue--ako-encoded.  With Prefix "ako-object-id--", user might think that this is uuid field of object (which is not the case). Also I think that prefixing instead of suffixing will make it difficult to understand format with <cluster-name> coming between prefix and encoded value (ako--encoded--<cluster-name>--encodedvalue>)

Reply:
@akshayhavile I meant the format to be like this"

cluster-name--ako-object-id--<SHA1> 

Does that make it better?

Reply:
If you want to simply get rid of `ako-object-id--" and say cluster-name--<SHA1> - that also is good enough in my opinion.